atlasdb-workload-server/src/main/java/com/palantir/atlasdb/workload/store/AtlasDbTransactionStoreFactory.java

```diff
@@ -138,6 +140,8 @@ public static AtlasDbTransactionStoreFactory createFromConfig(
                 .globalMetricsRegistry(metricsManager.getRegistry())
                 .globalTaggedMetricRegistry(metricsManager.getTaggedRegistry())
                 .runtimeConfigSupplier(atlasDbRuntimeConfig)
+                .defaultTimelockClientFactory(
+                        lockService -> TimeLockClient.createDefault(UnreliableTimeLockService.create(lockService), 100))
```

Comment:
not actionable: This is in workload so probably fine, though hard coding 100 here feels a bit magic